spec/integration/general_shopping_list_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'General Shopping List', type: :feature do
+  describe 'index page' do
+    before(:each) do
+      User.destroy_all
+      Food.destroy_all
+      Recipe.destroy_all
+      RecipeFood.destroy_all
+      user = User.create(name: 'user', email: 'user@gmail.com', password: 123_456)
+      food = Food.create(name: 'meat', measurement_unit: 'lib', price: 1, quantity: 3, user:)
```

Comment:
- [ ] Are you sure `user:` should stay like that? I'm sure you missed something or you wanted to put it but were not able to do it before I review it 😊.  Kindly complete the initialization.